packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -183,7 +185,11 @@
   flex-shrink: 0;
 }
 
-.dismissalOneline {
+.dismissOneline {
```

Comment:
i tweaked dismissalOneline to rootIsOneline (as the style actually applies to the root). The new dismissOneline is used.